projects/vdk-core/tests/functional/run/test_run_configuration.py

```diff
@@ -134,7 +143,7 @@ def run_job(self, context: JobContext) -> Optional[ExecutionResult]:
         runner = CliEntryBasedTestRunner(config_log)
         result: Result = runner.invoke(["run", util.job_path("simple-job")])
         cli_assert_equal(0, result)
-        # secrets that match config keys are overridden
-        assert config_log.config.get_value("log_exception_formatter") == "plain"
+        # secrets that match config keys but are configured using env variables are NOT overridden
+        assert config_log.config.get_value("log_exception_formatter") == "pretty"
```

Comment:
this looks to me like it is taking the property from the ini file and not env variable?
It should be over ridden by the env variable ?